aws-lambda-java-events-sdk-transformer/src/main/java/com/amazonaws/services/lambda/runtime/events/transformers/dynamodb/DynamodbStreamRecordTransformer.java

```diff
@@ -2,25 +2,38 @@
 
 import software.amazon.awssdk.services.dynamodb.model.StreamRecord;
 
+import java.util.Map;
+
 public class DynamodbStreamRecordTransformer {
 
     public static StreamRecord toStreamRecordV2(final com.amazonaws.services.lambda.runtime.events.models.dynamodb.StreamRecord streamRecord) {
-        return StreamRecord.builder()
+        StreamRecord.Builder streamRecordBuilder = StreamRecord.builder()
                 .approximateCreationDateTime(
                         streamRecord.getApproximateCreationDateTime().toInstant()
                 )
                 .keys(
                         DynamodbAttributeValueTransformer.toAttributeValueMapV2(streamRecord.getKeys())
                 )
-                .newImage(
-                        DynamodbAttributeValueTransformer.toAttributeValueMapV2(streamRecord.getNewImage())
-                )
-                .oldImage(
-                        DynamodbAttributeValueTransformer.toAttributeValueMapV2(streamRecord.getOldImage())
-                )
```

Comment:
Same as above (ternary operator). This would allow us to keep the function simple and minimal as opposed to breaking up the builder and having intermediate checks and operations.

Reply:
Extracted two local variables wit ternary operator and address the breaking builder